### PR TITLE
Inline write_cmpr_ulong

### PR DESCRIPTION
**Pull request by baldersheim**
@havardpe PR

### CODE REVIEW

File: vespalib/src/vespa/vespalib/data/slime/binary_format.cpp [hunk 1]
```diff
@@ -71,15 +90,13 @@ struct BinaryEncoder : public ArrayTraverser,
 };
 
 void
-BinaryEncoder::entry(size_t, const Inspector &inspector)
-{
+BinaryEncoder::entry(size_t, const Inspector &inspector) {
```
Comment (havardpe) [inline vespalib/src/vespa/vespalib/data/slime/binary_format.cpp:93]: why change formatting here?

File: vespalib/src/vespa/vespalib/data/slime/binary_format.cpp [hunk 2]
```diff
@@ -71,15 +90,13 @@ struct BinaryEncoder : public ArrayTraverser,
...
     encodeValue(inspector);
 }
 
 void
-BinaryEncoder::field(const Symbol &symbol, const Inspector &inspector)
-{
-    write_cmpr_ulong(out, symbol.getValue());
+BinaryEncoder::field(const Symbol &symbol, const Inspector &inspector) {
```
Comment (havardpe) [inline vespalib/src/vespa/vespalib/data/slime/binary_format.cpp:98]: why change formatting here?

File: vespalib/src/vespa/vespalib/data/slime/binary_format.cpp [hunk 3]
```diff
@@ -108,17 +125,17 @@ struct MappedSymbols {
...
         return (symbol.getValue() < symbol_mapping.size())
-            ? symbol_mapping[symbol.getValue()]
-            : symbol;
+               ? symbol_mapping[symbol.getValue()]
+               : symbol;
     }
 };
 
-template <bool remap_symbols>
+template<bool remap_symbols>
```
Comment (havardpe) [inline vespalib/src/vespa/vespalib/data/slime/binary_format.cpp:133]: why change formatting here?

File: vespalib/src/vespa/vespalib/data/slime/binary_format.cpp [hunk 4]
```diff
@@ -108,17 +125,17 @@ struct MappedSymbols {
...
 struct SymbolHandler {
     typedef typename std::conditional<remap_symbols, MappedSymbols, DirectSymbols>::type type;
 };
 
-template <bool remap_symbols>
+template<bool remap_symbols>
```
Comment (havardpe) [inline vespalib/src/vespa/vespalib/data/slime/binary_format.cpp:138]: why change formatting here?

File: vespalib/src/vespa/vespalib/data/slime/binary_format.cpp [hunk 5]
```diff
@@ -197,10 +212,9 @@ struct BinaryDecoder : SymbolHandler<remap_symbols>::type {
     }
 };
 
-template <bool remap_symbols>
+template<bool remap_symbols>
```
Comment (havardpe) [inline vespalib/src/vespa/vespalib/data/slime/binary_format.cpp:215]: why change formatting here?

File: vespalib/src/vespa/vespalib/data/slime/binary_format.cpp [hunk 6]
```diff
@@ -197,10 +212,9 @@ struct BinaryDecoder : SymbolHandler<remap_symbols>::type {
...
 Cursor &
-BinaryDecoder<remap_symbols>::decodeArray(const Inserter &inserter, uint32_t meta)
-{
+BinaryDecoder<remap_symbols>::decodeArray(const Inserter &inserter, uint32_t meta) {
```
Comment (havardpe) [inline vespalib/src/vespa/vespalib/data/slime/binary_format.cpp:217]: why change formatting here?

File: vespalib/src/vespa/vespalib/data/slime/binary_format.cpp [hunk 7]
```diff
@@ -210,10 +224,9 @@ BinaryDecoder<remap_symbols>::decodeArray(const Inserter &inserter, uint32_t met
     return cursor;
 }
 
-template <bool remap_symbols>
+template<bool remap_symbols>
```
Comment (havardpe) [inline vespalib/src/vespa/vespalib/data/slime/binary_format.cpp:227]: why change formatting here?

File: vespalib/src/vespa/vespalib/data/slime/binary_format.cpp [hunk 8]
```diff
@@ -210,10 +224,9 @@ BinaryDecoder<remap_symbols>::decodeArray(const Inserter &inserter, uint32_t met
...
 Cursor &
-BinaryDecoder<remap_symbols>::decodeObject(const Inserter &inserter, uint32_t meta)
-{
+BinaryDecoder<remap_symbols>::decodeObject(const Inserter &inserter, uint32_t meta) {
```
Comment (havardpe) [inline vespalib/src/vespa/vespalib/data/slime/binary_format.cpp:229]: why change formatting here?

**Review by havardpe — APPROVED** [review]
Some formatting changes seems unmotivated and inconsistent (tagged). Not sure if they are intentional or not.

**Comment by baldersheim** [comment]
Clion reformatted everything withing the anonymous namespace. I reverted most but not all obviously. Will undo the rest

File: vespalib/src/vespa/vespalib/data/slime/binary_format.cpp (hunk 1 above)
Comment (baldersheim) [reply vespalib/src/vespa/vespalib/data/slime/binary_format.cpp]: Fixed

File: vespalib/src/vespa/vespalib/data/slime/binary_format.cpp (hunk 2 above)
Comment (baldersheim) [reply vespalib/src/vespa/vespalib/data/slime/binary_format.cpp]: Fixed

File: vespalib/src/vespa/vespalib/data/slime/binary_format.cpp (hunk 3 above)
Comment (baldersheim) [reply vespalib/src/vespa/vespalib/data/slime/binary_format.cpp]: Fixed

File: vespalib/src/vespa/vespalib/data/slime/binary_format.cpp (hunk 4 above)
Comment (baldersheim) [reply vespalib/src/vespa/vespalib/data/slime/binary_format.cpp]: Fixed

File: vespalib/src/vespa/vespalib/data/slime/binary_format.cpp (hunk 5 above)
Comment (baldersheim) [reply vespalib/src/vespa/vespalib/data/slime/binary_format.cpp]: Fixed

File: vespalib/src/vespa/vespalib/data/slime/binary_format.cpp (hunk 7 above)
Comment (baldersheim) [reply vespalib/src/vespa/vespalib/data/slime/binary_format.cpp]: Fixed

File: vespalib/src/vespa/vespalib/data/slime/binary_format.cpp (hunk 8 above)
Comment (baldersheim) [reply vespalib/src/vespa/vespalib/data/slime/binary_format.cpp]: Fixed

**Pull request merged**